tools/imageregistry/setup_imageregistry.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+kubectl create ns rukpak-system || true
+
+# create self-signed certificate and authorization secrets for registry server
+
+mkdir -p /tmp/var/imageregistry/certs /tmp/var/imageregistry/auth
+openssl req -x509 -newkey rsa:4096 -days 365 -nodes -sha256 -keyout /tmp/var/imageregistry/certs/tls.key -out /tmp/var/imageregistry/certs/tls.crt -subj "/CN=docker-registry" -addext "subjectAltName = DNS:docker-registry"
+docker run --rm --entrypoint htpasswd registry:2.6.2 -Bbn myuser mypasswd > /tmp/var/imageregistry/auth/htpasswd
```

Comment:
Can an init container setup htpasswd as part of the registry pod using an emptyDir shared between the init container and the registry server container?

Reply:
It should be possible.   I'll try.

Reply:
It worked.  I pushed changes.